### PR TITLE
Operator sdk version

### PR DESCRIPTION
**Pull request by kevinearls**
## Which problem is this PR solving?
- Make sure we always use the correct version of the operator-sdk in builds.  Suggested in chat by @pavolloffay   I can open an issue if needed.

## Short description of the changes
- Added a Makefile target to install the operator-sdk, mostly copied from Pavol's equivalent change to the open-telemetry collector: https://github.com/open-telemetry/opentelemetry-operator/pull/728
-

### CODE REVIEW

**Comment by kevinearls** [comment]
Woops.  Need to rebase this.  Will reopen.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -12,80 +12,69 @@ This project is a regular [Kubernetes Operator](https://coreos.com/operators/)
 
 ### Installing the Operator SDK command line tool
 
-Follow the installation guidelines from [Operator SDK GitHub page](https://github.com/operator-framework/operator-sdk) or run `make install-sdk`.
+Follow the installation guidelines from [Operator SDK GitHub page](https://github.com/operator-framework/operator-sdk)
```
Comment (frzifus) [inline CONTRIBUTING.md:15]: Isnt this part of [this pr](https://github.com/jaegertracing/jaeger-operator/pull/1762) already?

Comment (kevinearls) [reply CONTRIBUTING.md:15]: Yes, that's why I closed this one.  ;-)

Comment (frzifus) [reply CONTRIBUTING.md:15]: ups now i see.

**Pull request closed without merging**